ostree/Docker/entrypoint.sh

```diff
@@ -68,4 +68,5 @@ export HOME=/home/"$username"
 
 sed -i 's/%sudo  ALL=(ALL:ALL) ALL/%sudo ALL=(ALL:ALL) NOPASSWD:ALL/' /etc/sudoers
 
-sudo -E -u "$username" "$@"
+"$@"
```

Comment:
Since it looks like this runs as root, on the user's PWD, won't generated artifacts also be owned by root?

Reply:
Added instruction to use chown to fix this (similar to the snapcraft Quick start documentation).